### ISSUE TITLE
Gumbel-Softmax Arxiv Docs Link Broken

### ISSUE THREAD

**Issue by ZaydH** [pytorch/pytorch#20297]
## 📚 Documentation

Docs for `torch.nn.functional.gumbel_softmax`. 

Function `gumbel_softmax` docstring entry for `Gumbel-Softmax distribution` has two Arxiv links:

    .. _Gumbel-Softmax distribution:
        https://arxiv.org/abs/1611.00712
        https://arxiv.org/abs/1611.01144

The official [PyTorch docs](https://pytorch.org/docs/stable/nn.html#gumbel-softmax) concatenate the two links together to make [https://arxiv.org/abs/1611.00712https://arxiv.org/abs/1611.01144](https://arxiv.org/abs/1611.00712https://arxiv.org/abs/1611.01144) which is clearly an invalid link.

*Note*: I am not sure how you would like this fixed.  It might be cleanest to separate it into two links. If you let me know, I can issue a pull request.

**Comment by ailzhang** [comment]
Hey @ZaydH , we are happy to take an PR to fix it! Thanks!

**Comment by KabirKwatra** [comment]
This issue can be closed
